incubator/volcano/config/kube-batch-ci.conf

```diff
@@ -0,0 +1,11 @@
+actions: "enqueue, reclaim, allocate, backfill, preempt"
```

Comment:
We need to maintain different copy for CI, since every action should be performed in CI, but for user it is not mandatory to run every actions, so we need to have seperate copy for CI.